aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -0,0 +1,18 @@
+---
+testnet: true
+remote_endpoint: ~
+framework_release:
+  bytecode_version: 6
+  git_hash: ~
+feature_flags:
+  enabled:
```

Comment:
So this will now be used to decide which features will be enabled in testnet?

Reply:
Yes. For the next release cycle.